lms/envs/common.py

```diff
@@ -2174,6 +2174,12 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.UserRateThrottle',
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'user': '50/minute',
```

Comment:
Just wondering how we came up with the value of `50/minute` here.  Also should this value be moved to configuration versus being managed here?